cnf-tests/testsuites/e2esuite/n3000/sriov-fec.go

```diff
@@ -0,0 +1,67 @@
+package n3000
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	fpgav1 "github.com/open-ness/openshift-operator/N3000/api/v1"
+	fecv1 "github.com/open-ness/openshift-operator/sriov-fec/api/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/client"
+)
+
+const (
+	sriovfecDeploymentName   = "sriov-fec-controller-manager"
+	daemonsetSriovPluginName = "sriov-device-plugin"
+	daemonsetSriovFecName    = "sriov-fec-daemonset"
+)
+
+var _ = Describe("Intel n3000: ", func() {
+	var sriovFecNodeConfigList = &fecv1.SriovFecNodeConfigList{}
+	var n3000NodeList = &fpgav1.N3000NodeList{}
+
+	BeforeEach(func() {
+		if !n3000Exists(n3000NodeList) {
+			Skip("Cluster doesn't have intel fpga n3000 nic")
+		}
+		if !sriovFecExists(sriovFecNodeConfigList) {
+			Skip("No sriovFec configuration on the cluster")
+		}
+	})
+
+	Context("SRIOV-FEC operator validation: ", func() {
+		// 39357
+		It("should have at least 1 sriov-fec nodeconfig", func() {
```

Comment:
shouldn't the test apply the config itself?

Reply:
`sriovfecnodeconfig` is create by default for each aplicable node after  `sriov-fec` operator is deployed
example:
$ oc get sriovfecnodeconfig -A

NAMESPACE                                         NAME                                      CONFIGURED
n3000-operators-resources   cnfdb5.clus2.t5g.lab.eng.bos.redhat.com   True